articles/search/search-indexer-field-mappings.md

```diff
@@ -378,6 +378,29 @@ When errors occur that are related to document key length exceeding 1024 charact
 ]
  ```
 
+<a name="toJsonFunction"></a>
+
+### toJson function
+ 
+This function converts a string into a formatted JSON object. This can be used for scenarios where the data source doesn't natively support compound / hierarchical data types, such as Azure SQL and then map it to complex fields.
```

Comment:
can you provide an example and explain what the characteristics of the string need to be in order for this to work?  I found this in the sql docs at https://learn.microsoft.com/en-us/sql/relational-databases/json/json-data-sql-server?view=sql-server-ver16:

{"id": 5, "info": {"name": "Jane", "surname": "Smith", "skills": ["SQL", "C#", "Azure"]}, "dob": "2005-11-04T12:00:00"}

so I think if I run toJsonFunction on that row, I would get this:

{
   "id":"5",
   "info":{
      "name":"Jane",
      "surname":"Smith",
      "skills":[
         "SQL",
         "C#",
         "Azure"
      ],
      "dob":"2005-11-04T12:00:00"
   }
}

Reply:
Lost all the indentation on the paste but it does resolve to well-formed json if you paste it into a linter.

Reply:
Great suggestion and I'll add this under the example section